tikv-client/src/test/java/com/pingcap/tikv/codec/TableCodecTest.java

```diff
@@ -25,7 +24,7 @@ private static TiTableInfo createTable() {
         .addColumn("c1", IntegerType.INT, true)
         .addColumn("c2", IntegerType.BIGINT)
         .addColumn("c3", DateTimeType.DATETIME)
-        .addColumn("c4", TimestampType.TIMESTAMP)
+        // .addColumn("c4", TimestampType.TIMESTAMP)
```

Comment:
Should we remove this commented code?

Reply:
currently only support write `java.lang.Long` to `TIMESTAMP`

Reply:
In that case, a TODO can add here or elsewhere associated with this problem.

Reply:
agree